hibernate-core/src/main/java/org/hibernate/type/Type.java

```diff
@@ -132,7 +132,7 @@
 	 * NOTE: The number of elements in this array matches the return from {@link #getColumnSpan}.
 	 *
 	 * @param mapping The mapping object :/
-	 * @todo Would be much much better to have this aware of Dialect once the service/metamodel split is done
+	 * TODO: Would be much much better to have this aware of Dialect once the service/metamodel split is done
```

Comment:
I think one of the reasons to use an annotation here is so that it doesn't end up in the generated Javadoc (because there is no doclet registered for the `@todo` annotation).

I don't think TODO's should end up in the Javadoc, so perhaps we should move this to a regular comment below instead.

Reply:
I reverted back the old `@todo`. Thanks.